include/seqan3/argument_parser/detail/format_parse.hpp

```diff
@@ -344,25 +345,20 @@ class format_parse : public format_base
      */
     template <std::Integral option_t>
     //!\cond
-        requires IStream<std::istringstream, option_t>
+        requires IStream<std::istringstream, option_t> &&
+                 requires (option_t & v, const char * ptr) { std::from_chars(ptr, ptr, v); }
```

Comment:
You are already constraining option_t to be integral, do you need this additional check?

Reply:
hmm if you define a type that models Integral it does not automatically work with from_chars does it? (But I actually noticed that this should reqire `arithmetic_concept` instead anyway.

Reply:
You cannot define a type that models std::Integral. std::Integral just covers the built-in types. Same for arithmetic

Reply:
I removed the check.